### DISCUSSION TITLE
Arxiv paper 8000-strong testing suite: where does it live? :)

### DISCUSSION THREAD

**Discussion by vadimkantorov** [tectonic-typesetting/tectonic#1141] [General]
Hi!

In https://github.com/tectonic-typesetting/tectonic/blob/release/CHANGELOG.md#tectonic-0141-2023-06-15 I noticed that the tectonic project seems to have fixed a 8000-strong arxiv paper testing suite.

Is this suite (or at least the arxiv url's for included papers) available somewhere[?](https://github.com/Mrmaxmeier/tectonic-on-arXiv) I'm also building a custom TexLive bundled binaries and it would be great to be able to report the compilation results on the same test battery as tectonic uses.

I checked https://github.com/tectonic-typesetting/tectonic/tree/master/tests, but it contains no README :( 

It would be awesome if this great testing suite was advertised somewhere for the commmunity / ecosystem to test on the same papers :)

Thank you!

**Comment by Mrmaxmeier** [comment]
Hi,

the arXiv tests are mostly undocumented but #397 includes some background. The test suite built from a tiny range of arXiv's [bulk data tarballs](https://info.arxiv.org/help/bulk_data_s3.html) that can be downloaded from S3 requester-pays buckets. If you're interested in the specific subset we're using you can find them here: https://tt.ente.ninja/datasets/
For implementation specifics you can look at https://github.com/Mrmaxmeier/tectonic-on-arXiv although the code is also criminally undocumented.
